pymc3/distributions/distribution.py

```diff
@@ -2,17 +2,21 @@
 import theano.tensor as tt
 from theano import function
 
+from . import transforms
```

Comment:
This creates a circular dependency. Not sure how it worked before. @fonnesbeck ?